tests/phpunit/payment-methods/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -121,6 +121,9 @@ public function set_up() {
 		update_option( WC_Stripe_Feature_Flags::LPM_ACSS_FEATURE_FLAG_NAME, 'yes' );
 		update_option( WC_Stripe_Feature_Flags::LPM_BACS_FEATURE_FLAG_NAME, 'yes' );
 		update_option( WC_Stripe_Feature_Flags::AMAZON_PAY_FEATURE_FLAG_NAME, 'yes' );
+
```

Comment:
Nit: Should we remove this line break to keep this `update_option()` next to the others?